src/styles/Fonts.scss

```diff
@@ -0,0 +1,2 @@
+$postText:-apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans',
```

Comment:
I think we should reuse fonts so the name should be more general (not specifici lie postText